src/build.ts

```diff
@@ -107,7 +107,11 @@ export default async function build(
       });
     }
 
-    console.log(`Build done. Available in ${buildPath}.`);
+    try {
+      const isInsideContainer = await fs.access('/.dockerenv').then(() => true).catch(() => false);
+      const outputPath = isInsideContainer ? './build' : buildPath;
+      console.log(`Build done. Available in ${outputPath}.`);
+    } catch (err) {}
```

Comment:
```suggestion
    const isInsideContainer = fs.existsSync('/.dockerenv')
    const outputPath = isInsideContainer ? './build' : buildPath;
    console.log(`Build done. Available in ${outputPath}.`);
```
Probably better to just to use `existsSync` as it's pretty much doing what's required.